README.md

```diff
@@ -17,6 +23,55 @@ python train.py --config-name <config>
 ```
 **--config-name** flag is used to specify config name, it needs to be in **configs** directory
 
+### Example config file
+```yaml
+hydra:
+  job:
+    chdir: False
+
+train:
+  task: "binary-classification"
+  dataset_name: "roszcz/ecg-segmentation-ltafdb" # huggingface dataset
+  batch_size: 128
+  num_workers: 8
+  lr: 3e-4
+  weight_decay: 0.01
+  num_epochs: 1
+  device: "cuda"
+  precision: "16-mixed" # not implemented yet
+  overfit_batches: 0
+
+paths:
+  save_ckpt_dir: "checkpoints" # directory where checkpoints will be saved
+  load_ckpt_path: null # if not None, specifies path to model state dict which will be loaded
```

Comment:
I'd say there's no reason to store a full config in readme - I think it's better to  just show a command that's guaranteed to work, i.e.:

```sh
# This would be L22 of this README
python train.py --config-name config-distribution-modelling
```
And mention that there are different configs available.